webapp/src/components/mobilitymap/InstructionsModal.js

```diff
@@ -0,0 +1,246 @@
+import api from "../../api";
+import React from "react";
+import Accordion from "react-bootstrap/Accordion";
+import Alert from "react-bootstrap/Alert";
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+
+const InstructionsModal = () => {
+  const [show, setShow] = React.useState(false);
+  const [accordionKey, setAccordionKey] = React.useState(null);
+  const [version, setVersion] = React.useState(null);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+  React.useEffect(() => {
+    if (!accordionKey) {
+      api.getUserAgent().then((r) => {
+        if (r.data) {
+          const browser = r.data;
+          if (browser.device === "Uknown") return;
```

Comment:
Our prettier should be auto-formatting to always have curly braces. If not please add curly braces- one line ifs can lead to very confusing and difficult to spot bugs.